### DISCUSSION TITLE
SqsAsyncClient Config error

### DISCUSSION THREAD

**Discussion by lebkset** [awspring/spring-cloud-aws#904] [Q&A]
I used spring boot 3.1.2 and spring cloud 3.0.2 when run  spring boot application then show error :
Could not generate CGLIB subclass of class software.amazon.awssdk.services.sqs.DefaultSqsAsyncClient: Common causes of this problem include using a final class or a non-visible

**Comment by jkuipers** [comment]
That class is indeed final, so cannot be subclassed. I guess the real question is why Spring feels like it should generate a CGLIB subclass for this type. Do you have something like an aspect whose pointcut expression matches this bean, perhaps?

**Comment by tomazfernandes** [comment]
@lebkset, does @jkuipers answer clarify your question?

**Comment by lebkset** [comment]
@jkuipers @tomazfernandes I want to confirm about configuration SqsAsyncClient such as bellow , is this true ?
```
@Configuration
public class SqsConfiguration {

  @Bean
  public SqsAsyncClient sqsAsyncClient() {
    return SqsAsyncClient.builder().build();
  }
}
```
